table/multiget_context.h

```diff
@@ -137,6 +137,9 @@ class MultiGetContext {
           sorted_keys_[iter]->lkey->user_key(),
           read_opts.timestamp == nullptr ? 0 : read_opts.timestamp->size());
       sorted_keys_[iter]->ikey = sorted_keys_[iter]->lkey->internal_key();
+      sorted_keys_[iter]->timestamp = (*sorted_keys)[begin + iter]->timestamp;
+      sorted_keys_[iter]->get_context =
+          (*sorted_keys)[begin + iter]->get_context;
```

Comment:
This is a semi-related bugfix, right?

Reply:
line 140 is a related fix, but line 142 is not. The comment says "copying", but original behavior of not shallow copying the get_context pointer is just misleading. It happens to work because it is nullptr after initialization.